docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/checkbox/checkbox.scss

```diff
@@ -5,7 +5,11 @@
 @import '../../00-base/mixins/form';
```

Comment:
do we need this mixin?

Reply:
Yes we are using the same mixin for the element CSS.